docs/edit/features.md

```diff
@@ -12,3 +12,29 @@ class Test_AwesomeFeature:
     def test_basic(self)
         assert P==NP
 ```
+
+Several key points:
+
+* Each new feature should be defined in [_features.py](/utils/_features.py). This consists of adding a feature in [Feature Parity Dashbaord](https://feature-parity.us1.prod.dog/), get the feature id and copying one of the already added features, changing the name and the feature id in the url, and the feature number. In this case we'd add
+
+```python
+
+    @staticmethod
+    def awesome_feature(test_object):
+        """
+        Awesome Feature for Awesomeness
+
+        https://feature-parity.us1.prod.dog/#/?feature=291
+        """
+        pytest.mark.features(feature_id=291)(test_object)
+        return test_object
+```
+
+* One class test one feature
+* One class can have several tests
+* Feature link to the [Feature Parity Dashbaord](https://feature-parity.us1.prod.dog/) is declared with `@features` decorators
+* Files can be nested (`tests/test_product/test_stuffs.py::Test_AwesomeFeature`), and how files are organized does not make any difference. Use you common sense, or ask on [slack](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X).
+
+## Skip tests
+
+See [skip-tests.md](/docs/edit/skip-tests.md)
```

Comment:
I think it can't hurt to have a link to the section since it's often that we need to skip tests for some tracers since they haven't implemented the feature.